### PR TITLE
Do not require configuration for every rule in yml file

### LINKED ISSUE #468: If the rule is disabled we should be able not to put extra configuration for it
Now when you want to disable a rule you need to put dummy  "configuration: {}" to the config file.
That is a bad practice and not user-friendly, need to enahnce it.

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Provided default value
* Edited all config files

This pull request closes #468 

The issue stated that disabled rules don't need configuration, but all configuration options have default values and it will work.

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: diktat-common/src/test/resources/test-rules-config.yml [hunk 1]
```diff
@@ -4,163 +4,116 @@
...
 - name: CONSTANT_UPPERCASE
   enabled: true
-  configuration: {}
 - name: ENUM_VALUE
   enabled: true
-  configuration: {}
 - name: EXCEPTION_SUFFIX
   enabled: true
-  configuration: {}
+  configuration: { }
```
Comment (akuleshov7) [inline diktat-common/src/test/resources/test-rules-config.yml:13]: ?

Comment (petertrr) [reply diktat-common/src/test/resources/test-rules-config.yml:13]: This is a yml file for tests, so I've kept different versions for configuration, which should be valid.

Comment (akuleshov7) [reply diktat-common/src/test/resources/test-rules-config.yml:13]: good, ok

**Pull request merged**